### ISSUE TITLE
Discontinuous selections give unintuitive animation results

### ISSUE THREAD

**Issue by jonstvns** [lukas-shawford/tiled-bulk-animations#3]
If you select two discontinuous tiles in the same row, you get a whole row of unexpected animated tiles. The resulting animations depend on the order you select the tiles.

Repro steps:
1. Open 
2. Select two tiles in the same row, but not touching (see Fig1)
3. Run the bulk animation tool with the parameters (r, 2, 100)
4. See at 3 animated tiles (see Fig2-4)
5. Open the animation editor
6. Notice the tile animations are not consecutive (see Fig2-4)

Expected results:
1. Only 2 animated tiles
2. Animations are consecutive for each animated tile

Related:

If you select tiles out of order, only some of the tiles receive animations.

For example:
1. Select tiles {1, 5, 2, 4} in that order
2. Run the animation tool with (r, 2, 100) 
3. You will get animations on tiles {1, 2, 3, 4}. No animation will be on tile 5. 

In other words, you only get animations on the tiles that lie between the first and last selected tiles. 

Test case files: [tilemap-example.zip](https://github.com/sergkr/tiled-bulk-animations/files/5123325/tilemap-example.zip)

Fig1
![image](https://user-images.githubusercontent.com/1030863/91164243-9d287500-e683-11ea-98f0-bea6e3c67d28.png)

Fig2
![image](https://user-images.githubusercontent.com/1030863/91164255-a0bbfc00-e683-11ea-8021-c92dd0c20e29.png)

Fig3
![image](https://user-images.githubusercontent.com/1030863/91164266-a580b000-e683-11ea-946a-d8ad6d05e7b2.png)

Fig4
![image](https://user-images.githubusercontent.com/1030863/91164275-a9143700-e683-11ea-9a08-829cf8f3f81a.png)

**Comment by lukas-shawford** [comment]
I just added support for non-rectangular / discontinuous selection regions in [v1.1](https://github.com/sergkr/tiled-bulk-animations/releases/tag/v1.1). Give it a try, and let me know if you see any issues.

I had to add an additional prompt for stride (number of tiles between each animation frame), as no assumptions can be made for irregular selection regions. I kept the prompt for rectangular regions also to keep things consistent and for added flexibility.
